### PR TITLE
Add DevTools-X

### PR DESCRIPTION
**Pull request by Sparkenstein**
This is the link to the project: [TITLE](URL)

- [ ] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [ ] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Padloc](https://github.com/padloc/padloc) - Modern, open source password manager for individuals and teams.
 - [Ghorbu Wallet](https://github.com/matthias-wright/ghorbu-wallet) - Cross-platform desktop HD wallet for Bitcoin.
 - [Stable Diffusion Buddy](https://github.com/breadthe/sd-buddy) - Desktop UI companion for the self-hosted Mac version of Stable Diffusion.
+- [DevTools-X](github.com/fosslife/devtools-x) - Cross-platform collection of development utils, all in one place
```
Comment (JonasKruckenberg) [inline README.md]:
```suggestion
- [DevTools-X](https://github.com/fosslife/devtools-x) - Cross-platform collection of development utilities, all in one place.
```

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]
Two things of note as well (do with this what you want):

1. `devtools` is not a really great bundle identifier, you should use a domain in reverse notation (ideally a domain you own) to avoid possible collisions. But even `fosslife.devtools` goes a long way to ensure you don't step on anyone's toes.

2. I know it's a devtools app and ppl take security less seriously there, but I can't see a good reason why your app needs 
   ```json
   "allowlist": {
      "all": true
   }
   ``` 
   Please take a moment to tighten that up.

**Comment by Sparkenstein** [comment]
will change the package identifier that makes sense. 

about allowList I kept it like that because I install npm dependencies very carefully. I especially avoid disputed packages like lodash and all. devtools only has handpicked deps. 
I understand deps of deps can be an issue. so I think I will fix this as well give me till weekend

**Comment by simonhyll** [comment]
@Sparkenstein We have updated the README structure a bit, please update your commit to reflect this change.

**Comment by Sparkenstein** [comment]
completely forgot about this. will close this one, and reopen again another.
I am assuming devtools-x goes in productivity section?

Edit or here https://github.com/tauri-apps/awesome-tauri#developer-tools

**Comment by simonhyll** [comment]
> completely forgot about this. will close this one, and reopen again another. I am assuming devtools-x goes in productivity section?
> 
> Edit or here https://github.com/tauri-apps/awesome-tauri#developer-tools

@Sparkenstein Good question, I think I'd place it under developer tools :)

**Pull request closed without merging**